README.md

```diff
@@ -193,7 +194,30 @@ In the following example, even though you call the `double-print-x-and-greet` fu
 
       (is (= "" (with-out-str (double-and-greet 2)))))))
 ```
-### 4. Combining different types of test doubles inside `with-doubles` macro.
+
+### 4. Making function calls throw exceptions
+You can use the `:throwing` option inside `with-doubles` macro to make functions all the calls to the functions throw exceptions with the message and data you want.
```

Comment:
"macro to make functions all the calls to the functions throw " => "macro to make all calls to the specified functions throw"

Reply:
Thanks André!